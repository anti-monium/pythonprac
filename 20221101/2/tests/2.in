A = Triangle((0,0), (0,1), (-1,0))
B = Triangle((1,-1), (0,2), (-2,0))
C = Triangle((3,-3), (-4,0), (0,3))
print(A in B, A in C, B in C, C in A)
if A < C:
	print('A - малепусик')
else:
	print('C - малепусик')
print(abs(A), abs(B), abs(C))
if A & B and A & C and B & C:
	print('Your my friend now turutututu')
if bool(B):
	print('B B B B B B B its B B B yo')
