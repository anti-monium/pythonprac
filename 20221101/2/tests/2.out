True True True False
A - малепусик
0.5 4.0 16.5
Your my friend now turutututu
B B B B B B B its B B B yo
